--- lrzip-0.16/Makefile.in.orig	2006-10-31 14:09:01.000000000 +0100
+++ lrzip-0.16/Makefile.in	2006-11-01 13:48:57.651380250 +0100
@@ -23,24 +23,7 @@
 .SUFFIXES:
 .SUFFIXES: .c .o
 
-OBJS= rzip.o runzip.o main.o stream.o util.o crc32.o\
-  LZMADecoder.o \
-  LZMAEncoder.o \
-  LZInWindow.o \
-  LZOutWindow.o \
-  RangeCoderBit.o \
-  InBuffer.o \
-  OutBuffer.o \
-  FileStreams.o \
-  Alloc.o \
-  C_FileIO.o \
-  CommandLineParser.o \
-  CRC.o \
-  String.o \
-  StringConvert.o \
-  StringToInt.o \
-  Vector.o \
-  LZMALib.o \
+OBJS= rzip.o runzip.o main.o stream.o util.o crc32.o
 
 
 # note that the -I. is needed to handle config.h when using VPATH
@@ -115,7 +98,7 @@
 	cd lzma && make && cd ../
 
 lrzip: $(OBJS)
-	$(CXX) $(CXXFLAGS) -o lrzip $(OBJS) $(LIBS)
+	$(CXX) $(CXXFLAGS) -o lrzip $(OBJS) -llzma $(LIBS)
 
 lrzip.1: lrzip.yo
 	yodl2man -o lrzip.1 lrzip.yo
--- lrzip-0.16/stream.c.orig	2006-10-31 14:17:12.000000000 +0100
+++ lrzip-0.16/stream.c	2006-11-01 13:48:39.182226000 +0100
@@ -20,7 +20,7 @@
 
 #include "rzip.h"
 #include "bzlib.h"
-#include "lzma/lzmalib.h"
+#include <lzmalib.h>
 
 #define CTYPE_NONE 3
 #define CTYPE_BZIP2 4
