--- lrzip-0.13/Makefile.in~	2006-10-30 22:44:08.091654462 +0200
+++ lrzip-0.13/Makefile.in	2006-10-30 22:45:28.753469087 +0200
@@ -5,7 +5,7 @@
 prefix=@prefix@
 exec_prefix=@exec_prefix@
 INSTALL_BIN=$(exec_prefix)/bin
-INSTALL_MAN=$(prefix)/man
+INSTALL_MAN=$(prefix)/share/man
 
 LIBS=@LIBS@ -llzma
 CC=@CC@
@@ -34,10 +34,10 @@
 man: rzip.1
 
 install: all
-	-mkdir -p ${INSTALL_BIN}
-	${INSTALLCMD} -m 755 lrzip ${INSTALL_BIN}
-	-mkdir -p ${INSTALL_MAN}/man1
-	${INSTALLCMD} -m 644 $(srcdir)/lrzip.1 ${INSTALL_MAN}/man1/
+	-mkdir -p $(DESTDIR)${INSTALL_BIN}
+	${INSTALLCMD} -m 755 lrzip $(DESTDIR)${INSTALL_BIN}
+	-mkdir -p $(DESTDIR)${INSTALL_MAN}/man1
+	${INSTALLCMD} -m 644 $(srcdir)/lrzip.1 $(DESTDIR)${INSTALL_MAN}/man1/
 
 liblzma: 
 	cd lzma/7zip/Compress/LZMA_Lib/ && make && cd ../../../../
