--- lrzip-0.13/configure.in~	2006-10-30 05:58:54.000000000 +0200
+++ lrzip-0.13/configure.in	2006-10-30 22:26:45.238189341 +0200
@@ -43,9 +43,9 @@
 
 
 AC_CHECK_LIB(bz2, BZ2_bzBuffToBuffCompress, , 
-        AC_MSG_ERROR([Could not find bz2 library - please install libbz2-dev]))
-AC_CHECK_LIB(lzo2, lzo1x_1_compress, , 
-        AC_MSG_ERROR([Could not find lzo2 library - please install liblzo2-dev]))
+        AC_MSG_ERROR([Could not find bz2 library - please install bzip2-devel]))
+AC_CHECK_LIB(lzo, lzo1x_1_compress, , 
+        AC_MSG_ERROR([Could not find lzo2 library - please install lzo-devel]))
 
 echo $ac_n "checking for errno in errno.h... $ac_c"
 AC_TRY_COMPILE([#include <errno.h>],[int i = errno],
--- lrzip-0.13/rzip.h~	2006-10-30 14:39:38.000000000 +0200
+++ lrzip-0.13/rzip.h	2006-10-30 22:28:47.240933191 +0200
@@ -34,8 +34,8 @@
 #include <stdlib.h>
 #include <string.h>
 #include <netinet/in.h>
-#include <lzo/lzoconf.h>
-#include <lzo/lzo1x.h>
+#include <lzoconf.h>
+#include <lzo1x.h>
 
 #ifdef HAVE_STRING_H
 #include <string.h>
