--- lrzip-0.13/stream.c~	2006-10-30 14:31:37.000000000 +0200
+++ lrzip-0.13/stream.c	2006-10-30 22:30:42.303522922 +0200
@@ -482,7 +482,7 @@
 			lzo_compress_buf(&sinfo->s[stream], &c_type, &c_len);
 		else if (control.flags & FLAG_BZIP2_COMPRESS)
 			bzip2_compress_buf(&sinfo->s[stream], &c_type, &c_len);
-		else fatal("Dunno wtf compression to use\!\n");
+		else fatal("Dunno wtf compression to use!\n");
 	}
 	//printf("ctype %d\n",c_type);
 
@@ -560,7 +560,7 @@
 		} else if (c_type == CTYPE_BZIP2) {
 			if (bzip2_decompress_buf(&sinfo->s[stream], c_len))
 				return -1;
-		} else fatal("Dunno wtf decompression type to use\!\n");
+		} else fatal("Dunno wtf decompression type to use!\n");
 	}
 
 	return 0;
